benchmarker/__main__.py

```diff
@@ -62,7 +62,10 @@ def main():
         else:
             result["device"] = result["platform"]["cpu"]["brand"]
         if args.flops:
-            result["problem"]["gflop_measured"] = get_counters(command)
+            if 'Intel(R)' in result["device"].split(" "):
+                result["problem"]["gflop_measured"] = get_counters(command)
+            else:
+                result["problem"]["gflop_measured"] = "N/A"
```

Comment:
Do we use `"N/A"` anywhere else? Don't we just omit those fields which we don't have? Won't this cause problems later, in the processing of arguments?!

Do you have to split the `result["device"]`? Can't you do it directly?

I would have just added here `if args.flops and 'Intel' in result["device"]:` and left it as is (+ get counters again doesn't return counters, right?)

Reply:
No, we don't use N/A anywhere else yet. We handle the processing of gflops_measured by checking if it is not a string